zuul.d/adoption.yaml

```diff
@@ -349,7 +349,7 @@
     voting: false
     timeout: 10800
     attempts: 1
-    nodeset: centos-9-multinode-rhel-9-2-crc-extracted-2-36-0-3xl-novacells
+    nodeset: centos-9-multinode-rhel-9-2-crc-extracted-2-39-0-3xl-novacells
```

Comment:
adding @bogdando fyi Lewis is working on bumping crc/ocp on the adoption jobs, including the novacells base here